test/start.js

```diff
@@ -433,3 +401,38 @@ test('should start the server listening on 0.0.0.0 when runing in docker', t =>
     t.strictEqual(fastify.server.address().address, '0.0.0.0')
   })
 })
+
+test('should start the server with watch options that the child process restart when directory changed', t => {
+  t.plan(5)
+
+  const tmpjs = path.resolve(baseFilename + '.js')
+  fs.writeFileSync(tmpjs, 'hello world')
+
+  setTimeout(function () {
+    const argv = [ '-p', '3000', '-w', './examples/plugin.js' ]
+    const fastifyEmitter = start.start(argv)
+
+    t.tearDown(() => {
+      fastifyEmitter.emit('close')
+      if (fs.existsSync(tmpjs)) {
+        fs.unlinkSync(tmpjs)
+      }
+    })
+
+    fastifyEmitter.on('start', () => {
+      t.pass('should receive start event')
+      setTimeout(function () {
+        t.pass('touch tmpjs')
+        touch.sync(tmpjs)
```

Comment:
it's looks like `touch.sync(tmpjs)` no trigger file change event or that event no caught by chokidar watch in Travis-CI.